docs/hil-bmi-commands.md

```diff
@@ -0,0 +1,41 @@
+# ESI Equivalency with HIL/BMI
+
+One of the goals of ESI is to replace [HIL/BML](https://github.com/CCI-MOC/hil). This document matches HIL/BMI commands with their OpenStack equivalents.
+
+* [Node Commands](#node)
+* [Network Commands](#network)
+
+## <a name="node"></a>Node Commands
+
+### Administrator
+
+* **hil node delete**: `openstack baremetal node delete`
+* **hil node nic delete/register**: ?? `openstack baremetal port create/delete`
```

Comment:
Does "??" mean "I'm not sure exactly what this HIL command does" or does it mean "I'm not entirely sure if this openstack command is an appropriate equivalent", or does it mean something else?

Reply:
Oh, here it means I'm not positive that the openstack command here is equivalent.

Reply:
VIFs - essentially, neutron ports - only need to be created when you attach a network to a node, at which point you'll:

* create a neutron port
* run 'baremetal vif attach' to attach that neutron port to a node; this process will update the baremetal port appropriately

So I think that baremetal port creation is enough to 'register' a nic; the rest will be handled when you attach a network to a node.